sites/mosul/configs/openmrs/initializer_config/ampathforms/F13-PNC.json

```diff
@@ -981,9 +981,7 @@
               "questionOptions": {
                 "rendering": "markdown"
               },
-              "value": [
-                "Scoring:(in\"i\":\"Sum of all points from answers above\") (answer is automatic sum of points from the 2 previous questions- when 3 or more points, automatic answer includes \"Please refer to Psychosocial counselling follow up)"
-              ],
```

Comment:
Please review the metadata file and revert this. This specific change affects the markdown which is meant to act as a guide to the users.

Reply:
This change is what came when I ran the form generator